onboarding/mentors.md

```diff
@@ -31,53 +71,83 @@ It's up to the mentoring pair to find what works best for them and to ensure the
     - Their manager
     - Their team lead
     - Artsy atlas, calendars, benefit websites, mail filters, Jira and Github management tools...
-  
 
-### Your first meeting
+**Your first meeting**
```

Comment:
#superminor but since this is targeted at mentors, perhaps this should be `The first meeting`